README.md

```diff
@@ -30,6 +30,60 @@ revision: HEAD
 
 See `conf/machine` for the complete list of supported devices.
 
+## Quick build
+Please refer to the [Yocto Project Quick Start Guide](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html) to set up your Yocto Project build environment.
+
+1. Clone this layer and its dependencies
+	```
+	git clone https://github.com/quic-yocto/meta-qcom-hwe.git -b main
+	git clone git://git.yoctoproject.org/meta-qcom -b master
+	git clone git://git.yoctoproject.org/poky -b master
+	```
+
+2. Initialize the build environment:
+	```
+	source poky/oe-init-build-env
+	```
+
+3. Add `meta-qcom` and `meta-qcom-hwe` layers to bblayers.conf:
+	```
+	bitbake-layers add-layer ../meta-qcom
+	bitbake-layers add-layer ../meta-qcom-hwe
+	```
+
+4. Set `MACHINE` in local.conf to one of the supported boards
+	```
+	MACHINE = "qcs6490-rb3gen2-core-kit"
+	```
+
+5. Set `INITRAMFS_IMAGE` in local.conf
```

Comment:
We have more settings in local.conf (in our KAS/CI config) such as the switch to systemd. I am not sure what we want to do with that.. but if someone builds using the instructions here, they will end up with an image without systemd for example, which we are not even testing.

Reply:
Maybe it would be better to just use kas for build, the necessary configuration is already defined and this way of using it is much simpler.

Reply:
right. that's a bit where I wanted to go. I am wondering what we really want to ask our users.. kas here is also new. the QCL 1.x (and LmP) use repo + setup-environment scripts. So we need to decide what we want long term for this branch. Are we even going to use poky here, or will we bring meta-qcom-distro into the build? Is this branch just a mechanism to test the standalone BSP layer, and we will get a 'main' branch in the qcom-manifest to do a full distro build?

Reply:
The idea for this branch alone is to be tested with poky only, we will have similar testing done against this branch in meta-qcom-distro, which will indeed cover more use cases and scenarios.

This is why I even removed the instructions here, as folks consuming this layer should really just use it as a BSP layer (but of course we should state what we support here). Maybe just point the instructions to kas and link the configuration used by kas for folks wanting to manually use this layer?

Reply:
I’ve made the updates as requested. Please review them.